### ISSUE TITLE
new release tag for changed dependencies?

### ISSUE THREAD

**Issue by joanwa** [masmu/pulseaudio-dlna#278]
Hey, 
just wondering since the last release is almost a year ago, would it make sense to tag a new release at least for the changed dependencies? More and more distros have packages for pulseaudio-dlna, and updating them will also take a while...

**Comment by masmu** [comment]
You are right. I was also thinking about that. This is going to happen soon!

**Comment by bdouxx** [comment]
Is there any planned/scheduled date for the next release?

**Comment by masmu** [comment]
Yeah. Hopefully in about 2 weeks. There is still a [branch](https://github.com/masmu/pulseaudio-dlna/tree/feature/dlna-improvements) where I made alot of DLNA improvements to also support _Raumfeld_ devices. And the company was so nice to provide me a testing device the next week. So after testing that, the release is ready to go.

In the meantime, everybody is welcome to also test that branch.

**Comment by nullr0ute** [comment]
What's the status of this?

**Comment by joanwa** [comment]
ping?

**Comment by prometheanfire** [comment]
ping again?

**Comment by prometheanfire** [comment]
and again?  I'd like to update the package on our distro...

**Comment by blueyed** [comment]
@masmu 
feature/dlna-improvements is merged by now, and I wondered about a new release for the DBus fixes - master works good for me, using a Raumfeld system currently.

**Comment by BridgeAR** [comment]
@masmu is there any update on this? Is there any blocker for the current master being released as 0.53?

**Comment by polyzen** [comment]
https://github.com/masmu/pulseaudio-dlna/issues/374#issuecomment-462038982
